release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -1028,6 +1032,18 @@ This update corrects the accepted label specification so that the plug-in import
 
 * Previously, when pressing a pool inside the block pools page, the final `Ready` phase persisted after deletion. Consequently, the pool was in the `Ready` state even after deletion. This update redirects users to the *Pools* page and refreshes the pools after detention. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1981396[*BZ#1981396*])
 
+[discrete]
+[id="ocp-4-10-dns-bug-fixes"]
+==== DNS
+
+* Previously, the DNS Operator did not enable the `prometheus` pug-in in the server blocks for custom upstream resolvers. Consequently, CoreDNS did not report metrics for upstream resolvers and only reported metrics for the default server block. With this update, the DNS Operator was changed to enable the `prometheus` plug-in in all server blocks. CoreDNS now reprts Prometheus metrics for custom upstream resolvers. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2020489[*BZ#2020489*])
+
+* Previously, an upstream DNS that provided a response greater than 512 characters caused an application to fail. This occurred because it could not clone the repository from GitHub because to DNS could not be resolved. With this update, the bufsize for KNI CoreDNS is set to 521 to avoid name resolutions from GitHub. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1991067[*BZ#1991067*])
+
+* When the DNS Operator reconciles its operands, the Operator gets the cluster DNS service object from the API to determine whether the Operator needs to create or update the service. If the service already exists, the Operator compares it with what the Operator expects to get in order to determine whether an update is needed. Kubernetes 1.22, on which {product-title} 4.9 is based, introduced a new `spec.internalTrafficPolicy` API field for services. The Operator leaves this field empty when it creates the service, but the API sets a default value for this field. The Operator was observing this default value and trying to update the field back to the empty value. This caused the Operator's update logic to continuously try to revert the default value that the API set for the service's internal traffic policy. With this fix, when comparing services to determine whether an update is required, the Operator now treats the empty value and default value for `spec.internalTrafficPolicy` as equal. As a result, the Operator no longer spuriously tries to update the cluster DNS service when the API sets a default value for the service's `spec.internalTrafficPolicy field`. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2002461[*BZ#2002461*])
```

Comment:
s/in order to/to

Reply:
Is there a way to break this down a little more? Just seems like a bit much, though I am not sure how you would really do that. I just tried to provide a suggestion but it seemed too wordy so idk where you could cut down.